sdk/boms/azure-sdk-bom/pom.xml

```diff
@@ -41,232 +38,227 @@
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-ai-formrecognizer</artifactId>
-        <version>3.1.8</version>
+        <version>3.1.9</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-ai-metricsadvisor</artifactId>
-        <version>1.1.0</version>
+        <version>1.1.1</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-ai-textanalytics</artifactId>
-        <version>5.1.6</version>
+        <version>5.1.7</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-communication-chat</artifactId>
-        <version>1.1.4</version>
+        <version>1.2.0</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-communication-common</artifactId>
-        <version>1.0.8</version>
+        <version>1.1.1</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-communication-identity</artifactId>
-        <version>1.1.6</version>
-      </dependency>
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-communication-networktraversal</artifactId>
```

Comment:
We removed this dependency from the BOM since networktraversal wanted to deprecate their GA. The package still exists on maven because there is no good way to deprecate a public package.

I have updated the minor version because of this. Also, I have not included it in the breaking changes log for I thought it would be best to not call attention to the thread.

Reply:
Here is the unique IP stats on the maven central
![image](https://user-images.githubusercontent.com/10713359/159082869-38b910ce-1a70-41a9-9e42-7613d9398838.png)

And total downloads
![image](https://user-images.githubusercontent.com/10713359/159082942-9749d539-4fab-4c2f-ac95-72033435b160.png)

Reply:
/cc: @srnagar , @JonathanGiles wanted to call this to your attention and make sure you agree.

Reply:
I'd at least call this out in the CHANGELOG

Reply:
Yeah I was feeling weird about updating the version but not giving any indication of why :)

Reply:
I have updated the changelog with the new information.